javascript/index.js

```diff
@@ -1,13 +1,40 @@
-import createSubscription from './modules/createSubscription';
 import reframeTime from './modules/reframeTime';
 import scrollConversationToBottom from './modules/scrollConversationToBottom';
-import setScrollListener from './modules/setScrollListener';
-import setSendMessageListener from './modules/setSendMessageListener';
+
+const conversationHistory = document.querySelector('.active-admin-chat__conversation-history');
+const sendMessageForm = document.querySelector('.active-admin-chat__send-message-form');
+let firstElement = null;
+let loadingMessages = false;
+let scrolling = false;
 
 document.addEventListener('DOMContentLoaded', () => {
   reframeTime();
   scrollConversationToBottom();
-  const channelSubscription = createSubscription();
-  setSendMessageListener(channelSubscription);
-  setScrollListener();
 });
+
+document.addEventListener('motion:render', () => {
+  reframeTime();
+
+  if (loadingMessages) {
```

Comment:
needed to have this variable to determine what triggered the `motion:render` was it a message from other person, or a message from myself?

Reply:
Is the goal here to scroll to the bottom when a new message was added?

Reply:
I guess I should rename the variable `loadingMessages` to `loadingOlderMessages`.

The feature is the following: if you scroll to the top of the chat until no more messages are visible in the frontend, older messages get automatically loaded and the user scrolling should be kept how it was. So, if this variable is `true` then after motion renders the updated chat with the older messages, I should scroll the chat to the message the user was seeing when the "re-rendering" for older messages happened.